test/attributes-to-props.test.js

```diff
@@ -186,6 +186,12 @@ describe('attributesToProps with HTML attribute', () => {
       expect(attributesToProps(attributes)).toEqual(props);
     }
   );
+
+  it('preserve value of option element', () => {
```

Comment:
Can you pluralize the verb?

```suggestion
  it('preserves value of option element', () => {
```